packages/react-chat/src/components/SystemResponse/SystemMessage.tsx

```diff
@@ -68,6 +69,7 @@ const SystemMessage: React.FC<SystemMessageProps> = ({ avatar, feedback, timesta
               .with({ type: MessageType.CAROUSEL }, (props) => (
                 <Carousel {...R.omit(props, ['type'])} containerRef={containerRef} controlsRef={controlsRef} />
               ))
+              .with({ type: MessageType.EXTENSION }, ({ payload }) => <ExtensionMessage extension={payload.extension} trace={payload.trace} />)
               .otherwise(() => null)}
           {feedback && <Feedback {...feedback} />}
         </List>
```

Comment:
these messages' containers should also stretch to 100% 
<img width="1406" alt="Screenshot 2024-02-29 at 12 32 42 PM" src="https://github.com/voiceflow/react-chat/assets/15104640/5b5c195c-34ab-4f3e-92b6-3d5758444c9b">